ROADMAP.md

```diff
@@ -1,164 +1,113 @@
-support for DSSE. in-toto-java and in-toto-rs were updated with support for ITEs
-5 and 6. We plan to continue working on our implementations to support new and
-existing ITEs. This will greatly aid us as we develop integrations with other
-projects and foster adoption in various organizations.
-
-In particular, we plan to work on the Python reference implementation to support
-ITE-6 attestations. It is the only implementation that currently has no support
-for attestations, and this is because of the stability guarantees we make for
-it. The last year has showed us that in-toto attestations are reaching stability
-and when ITE-6 is accepted, it will be time to support attestations in the
-reference implementation as well.
-
-Our work on in-toto implementations will continue to be in conjunction with the
-development of ITEs. Some exciting projects we expect to work on this year are:
-* supporting in implementations for the layouts that can verify attestations
-* adding ITE-9 reviewed predicates to one or more implementations
-* improving usability and simplifying the deployment of in-toto and TUF (ITE-2)
-  in software supply chains
-
-## Integrations
-
-Over the years, in-toto has been integrated into various other tools and
-systems. We plan to continue supporting their development and collaborating with
-other software supply chain security stakeholders. We are excited about the work
-we're doing with projects like Keylime, GUAC, and Sigstore, and we hope to talk
-about it more over the course of this year.
-
-Integrations we maintain ourselves will continue to get new features and
-support. Our Jenkins plugin recently received the capability to generate SLSA
-Provenance alongside its support for in-toto link metadata.
-
-## Community
-
-One of the efforts that has already started is a revamp of how the in-toto
-community is managed. A new starting point for all things in-toto was set up
-recently: https://github.com/in-toto/community. We believe this will help
-summarize the various in-toto projects and implementations in one place, as well
-as describe aspects of the community such as its governance, meeting cadence,
-contributing guidelines, and more.
-
-## Roadmap Review Schedule
-
-We have a new release schedule for our reviews as our roadmap now applies to the
-calendar year.
-
-- End of April 2023
-- End of August 2023
-- End of December 2023
-
-We will use these slots to release our roadmap reviews and depending on the
-development of various features, reviews may be accompanied by releases of our
-implementations.
\ No newline at end of file
+[Link to Roadmap 2022](/roadmap-reviews/2023/ROADMAP.md)_
+
+## Ongoing Work
+
+Cross-collaboration with many external groups and projects, including:
+
+- OpenSSF and groups
+  - SBOMit
+  - GitTUF
+  - Protobom
+- IETF
+  - Supply Chain Integrity, Transparency, and Trust (SCITT) Working Group
+- Package Repositories
+  - PyPi
+  - RubyGems
+
+## Recently Completed Work
```

Comment:
That's a good point. Since this roadmap is at a pretty macro-level (yearly or more), the intent to give unfamiliar folks a consolidated place to understand the activity of the projects. As far as making it explicit any suggestions?

Reply:
I wonder if it's as simple as making a note of the intent of this document at the top as well as in the repo README. That way we'd also have a pointer to the roadmap, which we currently lack. Surfacing it better would be good?

cc @in-toto/in-toto-steering-committee-itsc